packages/suite/src/hooks/wallet/useSendFormCompose.ts

```diff
@@ -273,8 +278,21 @@ export const useSendFormCompose = ({
             clearErrors(composeErrors);
         }
         // start composing
-        updateContext({ account, isLoading: true });
-    }, [state.account, state.isDirty, account, clearErrors, errors, updateContext]);
+        updateContext({
+            account,
+            excludedUtxos: getExcludedUtxos(account, state.feeInfo, targetAnonymity),
```

Comment:
this is part of the reason why I always prefer passing more than one param to function in object so it's named (i.e  `getExcludedUtxos({ account, dustLimit: state.feeInfo.dustLimit, targetAnonymity})`)
```suggestion
            excludedUtxos: getExcludedUtxos(account, state.feeInfo.dustLimit, targetAnonymity),
```

Reply:
Fixed in b9c0a769947a03f3e4b77c791951a6913240cb52. I did not change the params yet to be in line with other utils, but good point and let's discuss that on code improvements